README.md

```diff
@@ -103,18 +100,36 @@ require additional configurations applied to
 ### Prerequisites
 
 The following prerequisites are required to use the helm chart:
+- [Helm 3](https://helm.sh/docs/intro/install/) (Helm 2 is not supported) 
+- See https://github.com/kubernetes/helm for more information.
 
-- [Helm 3](https://helm.sh/docs/intro/install/) (Helm 2 is not supported)
-- [Kubernetes cluster](https://kubernetes.io/)
-
-- To send data to [Splunk Enterprise/Cloud](https://docs.splunk.com/Documentation/Splunk/8.2.2/Data/UsetheHTTPEventCollector)
-  - HEC Token
-  - HEC Endpoint
+#### To send data to Splunk Enterprise/Cloud:
```

Comment:
If we transform it to header, can we do the same for Splunk Observability Cloud for consistency?